### ISSUE TITLE
Thermos Server Crash when People try to log-in (thermos-crash)

### ISSUE THREAD

**Issue by lightning02** [CyberdyneCC/Thermos#434]
**Server Log:** https://www.dropbox.com/s/yqddtq1eaexzh69/latest.log?dl=0

**FML Log(s):** https://www.dropbox.com/s/iin5rajczeeif7f/fml-server-latest.log?dl=0

**Explanation of issue:** The client and the server side are synched (except for client side mods naturally), when people try to join in the server, however, the server crash and close, while the client doesn't have any problem.
I've tryed with both thermos build 55 and 57 and the issue is the same.
I've also tryed with a 1.7.10 normal forge server, it works without any problem.
http://imgur.com/a/3QvJZ

**How to recreate this issue:** To reproduce, I guess you can just use the same mods and eventually track it down...

**Mods Installed:** http://pastebin.com/ZT3UxCHP

**Plugins Installed:** No plugins installed

**Warmroast Report:** 

**Thermos Version:** build 55 and build 77 (tried both)

**Forge Version:** (1.7.10-1614)

**Comment by MegaFlary** [comment]
Same as #431.

**Comment by lightning02** [comment]
Yeah it is! :O
well, I've tryed to figured out how to modify the value, did you used Eclipse? how was you able to manage that? I'm not skilled with java at all, and I noticed that with Java decompiler I can only read without modify the file :l
I'm wasn't even able to find the source code of tfc, lol
Btw, do you mind to share the modified .class file with me eventually? >_<
I'm not sure if I'm able to edit that :c

**Comment by MegaFlary** [comment]
Source code is https://github.com/Deadrik/TFCraft

**Comment by lightning02** [comment]
Okay... I'll make some more try, thanks for the advice by the way :)

**Comment by lightning02** [comment]
Ok, since I'm a total noob with java I've just downloaded the tfc master at github, then modified the file TF_EntityPlayerMP as you suggested in the other thread-issue and finally built all in a new jar with the express .bat file in the master folder (a friend helped me with that btw)..
In the end I've replaced the new jar and tryed again.
The result is that the server no longer crash when people try to log-in, but they aren't able to log-in... this doesn't occur on normal forge server, only with thermos, and even with build 53...
Any other suggestion? :/
Should I re-open the issue since isn't solved?
here are the logs 
https://www.dropbox.com/s/2na9bclho1apw6y/fml-server-latest.log?dl=0
https://www.dropbox.com/s/dkci79rfd8n63wd/latest.log?dl=0
http://imgur.com/lzL3jIr
